parser/src/java/org/apache/hadoop/hive/ql/parse/AlterClauseParser.g

```diff
@@ -503,6 +503,8 @@ alterStatementSuffixExecute
     -> ^(TOK_ALTERTABLE_EXECUTE KW_EXPIRE_SNAPSHOTS KW_RETAIN $numToRetain)
     | KW_EXECUTE KW_DELETE KW_ORPHAN_FILES (KW_OLDER KW_THAN LPAREN (timestamp=StringLiteral) RPAREN)?
     -> ^(TOK_ALTERTABLE_EXECUTE KW_ORPHAN_FILES $timestamp?)
+    | KW_EXECUTE KW_RENAME KW_BRANCH sourceBranch=StringLiteral KW_TO targetBranch=StringLiteral
```

Comment:
why do we need `execute` here? simple rename should be enough
````
alter table ice01 rename branch 'source' to 'target';
````

Reply:
Agreed with @deniskuzZ .  
I think we need to reach an agreement on when to use this `execute` syntax.
IMHO, `execute` is good for some optimization & maintenance operation. I think TrinoDB also thought so:(https://trino.io/docs/current/connector/iceberg.html#alter-table-execute)

For basic operation, rename can use the same simple syntax like create/drop.

Reply:
thanx folks, I have changed the syntax & pushed